batch/test/test_batch.py

```diff
@@ -674,6 +674,27 @@ def test_verify_no_access_to_metadata_server(client):
     assert "Connection timed out" in j.log()['main'], (str(j.log()['main']), status)
 
 
+def test_submit_batch_in_job(client):
+    builder = client.create_batch()
+    bucket_name = get_user_config().get('batch', 'bucket')
+    script = f'''import hailtop.batch as hb
```

Comment:
Also, can we bump the instance version? This test is going to fail on old versions of the workers. Not sure it's worth trying to prevent the deploy failure.